pkg/scheduler/core/generic_scheduler.go

```diff
@@ -730,6 +730,9 @@ func PrioritizeNodes(
 				mu.Lock()
 				for i := range *prioritizedList {
 					host, score := (*prioritizedList)[i].Host, (*prioritizedList)[i].Score
+					if glog.V(10) {
+						glog.Infof("%v -> %v: %v, Score: (%d)", pod.Name, host, ext.Name(), score)
```

Comment:
pod name + namespace

Reply:
It is consistent with generic_scheduler.go#PrioritizeNodes(line 695). Maybe I change it together?